test/ClangImporter/Inputs/typedef-with-generic-param.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+@interface Foo <__covariant T>
+typedef void (^CompletionBlock)(T _Nullable result, NSError *_Nullable error);
```

Comment:
The problem here is not `typedef` itself but how it's used in the class method down the road.